extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/RasterDemSource.kt

```diff
@@ -157,7 +156,7 @@ class RasterDemSource(builder: Builder) : Source(builder.sourceId) {
      */
     get() {
       getPropertyValue<String?>("encoding")?.let {
-        return Encoding.valueOf(it.toUpperCase(Locale.US).replace('-', '_'))
```

Comment:
is this change intended?

Reply:
`toUpperCase` is deprecated in favour of `uppercase`